spire-flex/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: spire-flex
```

Comment:
The repo is already separate. Should the chart just be called 'spire' ?

Reply:
To avoid confusion with the SPIRE project and the other SPIRE helm chart, let's hold on to spire-flex for a little longer.  If you feel we need to revisit this, please create an issue so the item can be deferred.